cmd/util/cmd/common/storage.go

```diff
@@ -35,3 +38,62 @@ func InitStorageWithTruncate(datadir string, truncate bool) *badger.DB {
 
 	return db
 }
+
+type Storages struct {
+	Headers            storage.Headers
+	Guarantees         storage.Guarantees
+	Seals              storage.Seals
+	Index              storage.Index
+	Payloads           storage.Payloads
+	Blocks             storage.Blocks
+	Setups             storage.EpochSetups
+	EpochCommits       storage.EpochCommits
+	Statuses           storage.EpochStatuses
+	Results            storage.ExecutionResults
+	Receipts           storage.ExecutionReceipts
+	ChunkDataPacks     storage.ChunkDataPacks
+	Commits            storage.Commits
+	Transactions       storage.Transactions
+	TransactionResults storage.TransactionResults
+	Collections        storage.Collections
+}
+
+func InitStorages(db *badger.DB) *Storages {
```

Comment:
We already have https://github.com/onflow/flow-go/blob/1e0a30e7a9118aad96a706639e37ec955f562542/storage/util/testing.go#L16 and https://github.com/onflow/flow-go/blob/add3a38fb96ecfb09fc5b3cccc49704c27c253d6/cmd/scaffold.go#L70.

I think we should consolidate these, maybe we replace all these with a `storage.All` wrapper struct and `Init` function in the `storage` package.

Reply:
Good suggestion. I've made the `storage.All` wrapper struct and the `Init` function. The command line is using them. 

But it will be a lot of changes to refactor the tests, will do them in a different PR.